wlms/server.go

```diff
@@ -290,6 +381,28 @@ func CreateServerUsing(acceptedConnections chan ReadWriteCloserWithIp, db UserDb
 		clientSendingTimeout:   time.Minute * 2,
 		clientForgetTimeout:    time.Minute * 5,
 		messagesOut:            messagesOut,
+		relay:                  relay,
+		ipv4:                   "",
+		ipv6:                   "",
+	}
+	// Get the IP addresses of our domain
+	ips, err := net.LookupIP("widelands.org")
```

Comment:
Should this be configurable? Does this not make local testing impossible?

Reply:
Admittedly: For local testing I replaced this part with hard-coded IPs. I guess it would also work with locally faked DNS resolving but I haven't tested it.

Reply:
Ups, missed the first part. Do you want it to be configurable?
What we need here are not the IPs of the metaserver but of the relay server. Making it configurable and using the IPs also for the RPC connection would allow to move the relay to another server when required at some time in the future.